scripts/cancelBatch.py

```diff
@@ -0,0 +1,53 @@
+# Script that takes in an array of batch names and applies take a bulk action such as removing the unique id or canceling all tasks.
+# By default, this script makes 50 concurrent API calls.
+import scaleapi
+from scaleapi.exceptions import ScaleException
+
+import argparse
+from concurrent.futures import ThreadPoolExecutor
+
+# Change this for update concurrency
+MAX_WORKERS = 50 
+
+# globals
+task_count = 0
+task_errors = []
+tasks_cancelled = []
+
+def main():
+
+    args = get_args()
+    clearUniqueId = args.clear or False
+
+    client = scaleapi.ScaleClient(args.api_key)
+    preProcessedBatches = args.batches.split(",")
+    batches = [word.strip() for word in preProcessedBatches]
+    for batch in batches:
+
```

Comment:
Here we can try get_batch to check if batch name is valid and queryable from API, and skip if it's not a valid batch name.